item_list.php

```diff
@@ -31,26 +31,60 @@
   echo '<div>For now, in order to add items to auction, you need to create auction first then add the items from the page</div>';
   echo '<div class="row py-4 h-75 ">';
   foreach ($items_for_current_page as $item) {
-      echo '<div class="col-md-4 p-2">';
-      echo '<div class="card justify-content-between  align-content-around " style="height: 300px;">';
-      echo '<img class="card-img-top" src="' . $item[3] . '" alt="' . $item[1] . '" style="height: 150px; object-fit: cover;">';
-      echo '<div class="card-body">';
-      echo '<h5 class="card-title">' . $item[1] . '</h5>';
-      echo '<p class="card-text text-truncate">' . $item[2] . '</p>';
-      echo '<p class="card-text">' . $item[4] . '</p>';
-      echo '</div>';
-      echo '</div>';
-      echo '</div>';
+    echo '<div id="item-' . $item[0] . '" class="col-md-4 p-2">';
+    echo '<div class="card" style="min-height: 450px;">';
+    echo '<img class="card-img-top" src="' . $item[3] . '" alt="' . $item[1] . '" style="height: 200px; object-fit: cover;">';
+    echo '<hr class="w-100 mb-auto "></hr`>';
+    echo '<div class="card-body d-flex flex-column">';
+    echo '<h5 class="card-title">' . $item[1] . '</h5>';
+    echo '<p class="card-text text-truncate">' . $item[2] . '</p>';
+    echo '<span class="card-text badge badge-light w-50 py-2">' . $item[4] . '</span>';
+    echo '<hr class="w-100 mt-auto "></hr`>';
+    echo '<button class="w-50 ml-auto btn btn-warning" onclick="removeItem(' . $item[0] . ')">Remove Item</button>';
+    echo '</div>';
+    echo '</div>';
+    echo '</div>';
   }
   echo '</div>';
 
-  // pagination
-  echo '<nav aria-label="Page navigation example">';
-  echo '<ul class="pagination py-4">';
-  for ($i = 1; $i <= $total_pages; $i++) {
-      echo '<li class="page-item' . ($i == $current_page ? ' active' : '') . '"><a class="page-link" href="?page=' . $i . '">' . $i . '</a></li>';
-  }
-  echo '</ul>';
-  echo '</nav>';
-  echo '</div>';
-?>
+    // pagination
+    echo '<nav aria-label="Page navigation example">';
+    echo '<ul class="pagination py-4">';
+    for ($i = 1; $i <= $total_pages; $i++) {
+        echo '<li class="page-item' . ($i == $current_page ? ' active' : '') . '"><a class="page-link" href="?page=' . $i . '">' . $i . '</a></li>';
```

Comment:
Has anyone been able to test this pagination logic that was included in the starter code?